apps/web/.env.test

```
@@ -10,3 +10,4 @@ QUICK_CREATE_ENABLED=true
 CRON_SECRET=1234567890abcdef1234567890abcdef1234
 INITIAL_ADMIN_EMAIL=initial.admin@rallly.co
 SMTP_REJECT_UNAUTHORIZED=false
+RATE_LIMIT_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RATE_LIMIT_ENABLED key should go before the SECRET_PASSWORD key

(UnorderedKey)